week-3/B-array-some/exercise.js

```diff
@@ -16,6 +16,9 @@ var students = ["Islam", "Lesley", "Harun", "Rukmini"];
 var mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
 var pairs = pairsByIndex.map(function(indexes) {
+  if (indexes === null) {
+    process.exit(1);
```

Comment:
This does work 🍰! But lets say you had to solve it using the `.some()` method. How would you do that?